docs/additional-functionality/rapids-shuffle.md

```diff
@@ -290,23 +290,39 @@ In this section, we are using a docker container built using the sample dockerfi
     | 3.0.3      | com.nvidia.spark.rapids.spark303.RapidsShuffleManager    |
     | 3.0.4      | com.nvidia.spark.rapids.spark304.RapidsShuffleManager    |
     | 3.1.1      | com.nvidia.spark.rapids.spark311.RapidsShuffleManager    |
+    | 3.1.1 CDH  | com.nvidia.spark.rapids.spark311cdh.RapidsShuffleManager |
     | 3.1.2      | com.nvidia.spark.rapids.spark312.RapidsShuffleManager    |
     | 3.1.3      | com.nvidia.spark.rapids.spark313.RapidsShuffleManager    |
-    | 3.2.0      | com.nvidia.spark.rapids.spark320.RapidsShuffleManager    |
-
-2. Recommended settings for UCX 1.10.1+
-```shell
-...
---conf spark.shuffle.manager=com.nvidia.spark.rapids.spark301.RapidsShuffleManager \
---conf spark.shuffle.service.enabled=false \
---conf spark.executorEnv.UCX_TLS=cuda_copy,cuda_ipc,rc,tcp \
---conf spark.executorEnv.UCX_ERROR_SIGNALS= \
---conf spark.executorEnv.UCX_RNDV_SCHEME=put_zcopy \
---conf spark.executorEnv.UCX_MAX_RNDV_RAILS=1 \
---conf spark.executorEnv.UCX_MEMTYPE_CACHE=n \
---conf spark.executorEnv.UCX_IB_RX_QUEUE_LEN=1024 \
---conf spark.executor.extraClassPath=${SPARK_CUDF_JAR}:${SPARK_RAPIDS_PLUGIN_JAR}
-```
+
+2. Settings for UCX 1.10.1+:
+
+    Minimum configuration:
+
+    ```shell
+    ...
+    --conf spark.shuffle.manager=com.nvidia.spark.rapids.spark301.RapidsShuffleManager \
```

Comment:
do we need comment saying change to correct version?  or make it generic <RapidsShuffleManager version from above table>

Reply:
yes let me phrase that in

Reply:
I think I am handling now with this change: https://github.com/NVIDIA/spark-rapids/pull/2994/commits/42deb5db0a0b13f90d50a0fb5bbdfb4dccf5ad1b.